utility/session.go

```diff
@@ -106,12 +106,14 @@ func (s *session) findClosestYFishermen() []*coreTypes.Actor {
 // Q) why do we hash to find a newKey between every actor selection?
 // A) pseudo-random selection only works if each iteration is re-randomized
 //    or it would be subject to lexicographical proximity bias attacks
-func (s *session) pseudoRandomSelection(orderedListOfPublicKeys []string, numberOfActorsInSession int) []*coreTypes.Actor {
-	// IMPORTANT:
-	// THIS IS A DEMONSTRABLE FUNCTION THAT WILL NOT BE IMPLEMENTED AS SUCH
-	// IT EXISTS IN THIS COMMIT PURELY TO COMMUNICATE THE EXPECTED BEHAVIOR
-	return nil
-}
+/*
```

Comment:
Was this error linting on unused?

Reply:
@Olshansk yeah this was unused - before I found out about `//nolint` so I can add it back in?

Reply:
Yea, let's add it back with a `not used but included for future implementation reference`